projects/plugins/jetpack/class.jetpack.php

```diff
@@ -306,8 +306,12 @@ class Jetpack {
 	/**
 	 * Plugins for which we turn off our Facebook OG Tags implementation.
 	 *
-	 * Note: All in One SEO Pack, All in one SEO Pack Pro, WordPress SEO by Yoast, and WordPress SEO Premium by Yoast automatically deactivate
-	 * Jetpack's Open Graph tags via filter when their Social Meta modules are active.
+	 * Note: the following plugins
+	 * automatically deactivate Jetpack's Open Graph tags via filter when their Social Meta modules are active:
```

Comment:
Line wrapping is weird.
```suggestion
         * Note: the following plugins automatically deactivate Jetpack's Open
         * Graph tags via filter when their Social Meta modules are active:
```